docs/workflow/testing/libraries/testing-android.md

```diff
@@ -22,7 +22,7 @@ Android SDK and NDK can be automatically installed via the following script:
 #!/usr/bin/env bash
 set -e
 
-NDK_VER=r23c
+NDK_VER=r27c
```

Comment:
Should we update our codespaces container definition? (I can do it, just want to make sure its the right thing to do)

 https://github.com/dotnet/runtime/blob/2a19c9d8bf7e1fba434987be3e45b607dbf07c50/.devcontainer/android/Dockerfile#L31

Reply:
Yes we should update the codespaces, we have upgraded the NDK version to 27c on the CI. This is also true about .NET 8.